morituri/program/flac.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import check_call, CalledProcessError
+
+import logging
+logger = logging.getLogger(__name__)
+
+FLAC = 'flac'
+
+
+def encode(infile, outfile):
+    """
+    Encodes infile to outfile, with flac.
+    Uses '-f' because morituri already creates the file.
+    """
+    try:
+        check_call(['flac', '--totally-silent', '--verify', '-o', outfile,
+                    '-f', infile])
```

Comment:
None of the options I mentioned alter the audio from the rip, and will result in a perfect copy when decoded.

The only option that could possibly alter the data would be --apply-replaygain-which-is-not-lossless, which pretty much explains itself and is a decoding-only option.

As for your function, yes, people who are already used to bash can do that, but think about the people who have recently gotten into Linux (or, in case of Windows or Mac users, who are inexperienced with command-line tools).